.github/workflows/pr-ci.yml

```diff
@@ -73,13 +103,32 @@ jobs:
         registry: ghcr.io/${{ github.repository_owner }}
         username: ${{ github.event.pull_request.user.login }}
         password: ${{ secrets.GHCR_PR_TOKEN }}
-      if: github.repository_owner == 'cryostatio'
-    - name: Comment test image link
-      uses: thollander/actions-comment-pull-request@v1
+    - name: store image amd64
+      id: amd64_image
+      run: echo "image=${{ steps.push-to-ghcr.outputs.registry-path }}" >> "$GITHUB_OUTPUT"
+      if: matrix.arch == 'amd64'
+    - name: store image arm64
```

Comment:
We don't need to specify separate steps (with `if`) for each arrch. This should work?

```yaml
      id: amd64_image
      run: echo "${{ matrix.arch }}_image=${{ steps.push-to-ghcr.outputs.registry-path }}" >> "$GITHUB_OUTPUT"
```

Also, u need to map job output to step output.

https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs